### DISCUSSION TITLE
How to remove/add item from array using API Rest ?

### DISCUSSION THREAD

**Discussion by tadamtech** [supabase/supabase#9002] [Questions]
Hello, 

I would like to perform HTTP API calls to add or remove items (ids) from an array. 

I have a table names USERS. Column name is ID_TO_SHOW and store arrays ["A", "B", "C", "n"]. I want to remove item "B". How can I do this with only one API Call using url parameters ?

Thank you in advance for you help !!! :)

**Comment by GaryAustin1** [comment]
You will need to write an rpc function to do this in one call…
https://github.com/supabase/supabase/discussions/2771
There were a couple of more examples searching on array remove here.
